core/src/main/scala/org/apache/spark/util/collection/ExternalSorter.scala

```diff
@@ -113,11 +114,21 @@ private[spark] class ExternalSorter[K, V, C](
     if (shouldPartition) partitioner.get.getPartition(key) else 0
   }
 
+  private val metaInitialRecords = conf.getInt("spark.shuffle.sort.metaInitialRecords", 256)
```

Comment:
I also wonder if we should just hard code this one. How do you envision users wanting to tune this?